index/Cargo.toml

```diff
@@ -15,6 +18,9 @@ hex = { version = "0.4.3", default-features = false }
 pink-extension = { version = "0.1", default-features = false, features = ["ink-as-dependency"] }
 pink-web3 = { git = "https://github.com/Phala-Network/pink-web3.git", branch = "pink", default-features = false, features = ["pink", "signing"]}
 xcm = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.29", default-features = false }
+ink_storage = { version = "3", default-features = false }
```

Comment:
In a previous PR I deliberately remove these ink! dependencies, is there a way we can do without them?